Chap_Terms.tex

```diff
@@ -20,23 +20,30 @@ \chapter{PMIx Terms and Conventions}
 The following terminology is used throughout this document:
 
 \begin{itemize}
-\item \declareterm{session} refers to a pool of resources with a unique identifier (a.k.a., the \emph{session ID}) assigned by the \ac{WLM} that has been reserved for one or more users. Historically, \ac{HPC} sessions have consisted of a static allocation of resources - e.g., a block of nodes assigned to a user in response to a specific request and managed as a unified collection. However, this is changing in response to the growing use of dynamic programming models that require on-the-fly allocation and release of system resources. Accordingly, the term \emph{session} in this document refers to a potentially dynamic entity, perhaps comprised of resources accumulated as a result of multiple allocation requests that are managed as a single unit by the \ac{WLM}.
 
-\item \declareterm{job} refers to a set of one or more \emph{applications} executed as a single invocation by the user within a session with a unique identifier (a.k.a, the \emph{job ID}) assigned by the \ac{RM} or launcher. For example, the command line ``\textit{mpiexec -n 1 app1 : -n 2 app2}'' generates a single \ac{MPMD} job containing two applications. A user may execute multiple \emph{jobs} within a given session, either sequentially or in parallel.
+\item \declareterm{session} 
+refers to a set of resources assigned by the \ac{WLM} that has been 
+reserved for one or more users. 
+A session is identified by a \emph{session ID} that is  
```

Comment:
"at least one user". Perhaps this would be better to make it clear that at implementation is not required to support multiple users within a session, for example, if the WLM does not permit it.